conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/LorreLogging.scala

```diff
@@ -2,33 +2,56 @@ package tech.cryptonomic.conseil.indexer
 
 import com.typesafe.scalalogging.Logger
 import tech.cryptonomic.conseil.BuildInfo
-import tech.cryptonomic.conseil.common.config.LorreConfiguration
 import tech.cryptonomic.conseil.common.config.Platforms.{BlockchainPlatform, PlatformConfiguration}
 import tech.cryptonomic.conseil.common.io.MainOutputs.{showDatabaseConfiguration, showPlatformConfiguration}
+import tech.cryptonomic.conseil.indexer.config.LorreConfiguration
 
-/** Defines what to print when starting Lorre */
-trait LorreMainOutput {
+import scala.concurrent.duration.{Duration, NANOSECONDS}
 
-  /** we need to have a logger */
-  protected[this] def logger: Logger
+trait LorreLogging {
```

Comment:
Personally I'd choose a less generic term.

*Logging says nothing about what the trait is about, and anything else related to logging might reuse the term in the future and create confusion on each module's responsibility

Reply:
I've split that into two different traits (cause they are also used separately), so method `display*` are in `LorreInfoLogging` and the first one is in `LorreProgressLogging`. Would that work?